site/docs-md/apis/camera/index.md

```diff
@@ -25,7 +25,7 @@ This API requires the following permissions be added to your `AndroidManifest.xm
 <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 ```
 
-The first permission is for Camera access, and the storage permissions are for reading/saving photo files.
+The storage permissions are for reading/saving photo files. Note that `android.permission.CAMERA` is not required.
```

Comment:
I removed the CAMERA permission the other day, so not sure if this note makes sense

Reply:
added it on the idea that people will be surprised that Camera permission isn't needed. I will remove

Reply:
Maybe it would confuse them after all :)

Reply:
Yeah, I forgot to remove that part, so that part of your change was OK, just not the note